utilities/proto/gen-options.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+#set -x
+
+function wrong_dir {
+  echo "Run from root directory"
+  exit 1
+}
+
+test -d common/cypherock-common/proto || wrong_dir
+
+# Make sure all directories exist for the proto-options
+test -d common/proto-options/ || mkdir common/proto-options/
+test -d common/proto-options/manager || mkdir common/proto-options/manager
+test -d common/proto-options/btc || mkdir common/proto-options/btc
+test -d common/proto-options/evm || mkdir common/proto-options/evm
```

Comment:
make this dynamic. so new folder addition will not break the script

Reply:
fixed